.travis.yml

```diff
@@ -1,21 +1,37 @@
+dist: bionic
 language: php
 
-php:
-  - 7.3
-  - 7.2
-
 env:
-  - PREFER_LOWEST=""
-  - PREFER_LOWEST="--prefer-lowest"
+  global:
+    - SETUP=stable
 
-jobs:
+matrix:
+  fast_finish: true
   include:
-    - script: composer lint
+    - php: 7.2
+    - php: 7.2
+      env: SETUP=lowest
+    - php: 7.3
+    - php: 7.3
+      env: SETUP=lowest
+    - php: 7.4
+    - php: 7.4
+      env: SETUP=lowest
+    - php: 7.4
```

Comment:
Ideally, we should be linting on 7.2, since that's the minimum supported version. But fuck it, sometimes you live life on the edge.